models/pathology_nuclei_segmentation_classification/large_files.yml

```diff
@@ -0,0 +1,5 @@
+large_files:
+  - path: "models/model.pt"
+    url: "https://drive.google.com/uc?id=1Auwm8YrQCx9EAX_oViNTlJ_JgBb-wbzZ"
```

Comment:
Can you update this model to have `model` key and re-upload?
```python
import torch
model = torch.load('model.pt')
model['model'] = model['net']
del model['net']
torch.save(model, 'model.pt')
```

Then can you also update the `hash_val` please? 
```
md5sum model.pt
```

Reply:
Have updated it, thanks!